upa/src/tool/aggregator.ts

```diff
@@ -210,31 +171,55 @@ const submitAggregatedProof = command({
     estimateGas: options.estimateGas(),
     dumpTx: options.dumpTx(),
     maxFeePerGasGwei: options.maxFeePerGasGwei(),
+    dumpArguments: option({
+      type: optional(string),
+      long: "dump-arguments",
+      description: "Write a JSON representation of the arguments to a file",
+    }),
     calldataFile: option({
       type: string,
       long: "calldata-file",
       short: "p",
       description: "Proof file",
     }),
-    proofIdsFile: option({
-      type: string,
-      long: "proof-ids-file",
-      short: "i",
-      description: "file with JSON list of proofIds to be aggregated",
-    }),
-    onChainSubmissionProofFiles: multioption({
+    submissionFiles: multioption({
       type: array(string),
-      long: "submission-proof-file",
+      long: "submission",
       short: "s",
-      defaultValue: () => [],
-      description: "submission proof file(s)",
+      description: "on-chain submission files",
     }),
-    offChainSubmissionMarkersFile: option({
-      type: string,
-      long: "submission-markers-file",
-      short: "s",
-      defaultValue: () => "",
-      description: "Unpacked submission markers file containing a boolean[]",
+    offset: option({
+      type: number,
+      long: "offset",
+      short: "",
+      defaultValue: () => 0,
+      description: "skip proofs in first on-chain submission (default: 0)",
+    }),
+    finalCount: option({
+      type: optional(number),
+      long: "final-count",
+      short: "f",
+      description:
+        "include only leading proofs from final final submission" +
```

Comment:
```suggestion
        "include only leading proofs from final submission" +
```

Reply:
I'm wasn't sure what this meant from just the description text

Reply:
fixed.
It's for when the last submission in the list is only partially included in the aggregated batch.  Any suggestions for better wording?